tests/test_upload.py

```diff
@@ -58,6 +59,99 @@ def test_successful_upload(make_settings, capsys):
     assert captured.out.count(NEW_RELEASE_URL) == 1
 
 
+def test_successful_upload_add_gpg_signature(make_settings, capsys, monkeypatch):
```

Comment:
No longer necessary.

```suggestion
def test_successful_upload_add_gpg_signature(make_settings, monkeypatch):
```

Reply:
Fixed in https://github.com/pypa/twine/pull/622/commits/458c42c2811559ba175f8d2a4bd60850ff59e75d